_config.yml

```
@@ -134,17 +134,42 @@ keep_files:
 
 # Plug-ins
 plugins:
+  - jekyll-archives
+  - jekyll-feed
+  - jekyll-sitemap
   - jekyll-email-protect
   - jekyll-github-metadata
-  - jekyll-feed
   - jekyll-paginate-v2
   - jekyll/scholar
   - jekyll-twitter-plugin
   - jemoji
 
+# Sitemap settings
+defaults:
+  -
+    scope:
```

nit: please change to the following:
```suggestion
  - scope:
```